snapshot/cli/cli.go

```diff
@@ -119,3 +128,51 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 	fmt.Println(id)
 	return nil
 }
+
+type ingestDirCommand struct {
+	dir string
+}
+
+func (c *ingestDirCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ingest_dir",
+		Short: "ingests a directory into the repo in cwd",
+	}
+	cmd.Flags().StringVar(&c.dir, "dir", "", "dir to ingest")
+	return cmd
+}
+
+func (c *ingestDirCommand) run(db snapshot.DB, _ *cobra.Command, _ []string) error {
+	id, err := db.IngestDir(c.dir)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(id)
+	return nil
+}
+
+type catCommand struct {
+	id string
+}
+
+func (c *catCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cat",
+		Short: "catenate file from an FSSnapshot to stdout",
```

Comment:
Right, but that's concatenate? I've never seen catenate before, just commenting that it may look weird to some.